packages/components/src/FormField/FormField.tsx

```diff
@@ -142,180 +147,179 @@ export interface FormFieldProps {
    * Callback to get the the status and message when validating a field
    * @param messages
    */
-  onValidation?(messages: ValidationProps[]): void;
+  onValidation?(messages?: ValidationProps[]): void;
 }
 
-export const FormField = React.forwardRef(
-  (
+export const FormField = React.forwardRef(FormFieldInternal);
+
+function FormFieldInternal(
+  {
+    align,
+    children,
+    defaultValue,
+    disabled,
+    errorMessage,
+    inline,
+    invalid,
+    max,
+    maxLength,
+    min,
+    name,
+    onFocus,
+    onBlur,
+    onChange,
+    onValidate,
+    onValidation,
+    placeholder,
+    readonly,
+    required,
+    rows,
+    size,
+    type = "text",
+    value,
+    validations = [],
+  }: FormFieldProps,
+  ref:
+    | Ref<HTMLInputElement>
+    | Ref<HTMLTextAreaElement>
+    | Ref<HTMLSelectElement>,
+) {
+  const [hasMiniLabel, setHasMiniLabel] = useState(
+    defaultValue || value ? true : false,
+  );
+  const identifier = uuid.v1();
+  const Wrapper = inline ? "span" : "div";
+
+  useEffect(() => {
+    /* DEPRECATED */
+    const status = errorMessage ? "fail" : "pass";
```

Comment:
I'm being dumb but, I think I have a question about this.

Reply:
What is the question?